ddp-workspace/projects/ddp-dsm-ui/src/app/file-download/file-download.component.html

```diff
@@ -0,0 +1,38 @@
+<ng-container>
```

Comment:
Wondering why there are so many `ng-container`-s here? 
Yu can get rid of all of them, like:
![image](https://user-images.githubusercontent.com/7396837/180981212-fa1f7d38-8215-4578-8c48-fe90d6d318e5.png)